### PR TITLE
prepwork-arrays

### PR DESCRIPTION
**Pull request by kbao24**
js-arrays

### CODE REVIEW

File: prep-javascript-arrays/main.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+const thirdElement = languages[2];
+
+console.log('the 3rd element of the array is', thirdElement);
+
+const length = languages.length;
+
+console.log('The length of the array is', length);
+
+const lastElement = languages[5];
```
Comment (shawnkost) [inline prep-javascript-arrays/main.js]: Rather than using `5` directly, you should be subtracting 1 from the `length` property of  `languages`

**Review by shawnkost — CHANGES_REQUESTED** [review]
Looking good for the most part Kevin! Just one small change here

File: prep-javascript-arrays/main.js (hunk 1 above)
Comment (kbao24) [reply prep-javascript-arrays/main.js]: My bad, I had written that in my quiz answers, but forgot to use it in the actual code. Thanks!

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**